front/src/applications/osrd/consts.ts

```diff
@@ -141,4 +141,5 @@ export interface OsrdConfState {
   featureInfoClick: { displayPopup: boolean; feature?: Feature; coordinates?: number[] };
   gridMarginBefore: number;
   gridMarginAfter: number;
+  standardStdcmAllowance: any; // We wait for auto generated types
```

Comment:
To be replaced when the openapi types are updated

Reply:
Agree comment for that